apache-rat-core/src/main/java/org/apache/rat/license/LicenseSetFactory.java

```diff
@@ -220,5 +221,5 @@ public static ILicense search(ILicense target, SortedSet<ILicense> licenses) {
         SortedSet<ILicense> part = licenses.tailSet(target);
         return (!part.isEmpty() && part.first().compareTo(target) == 0) ? part.first() : null;
```

Comment:
Can target be null? Should we use part().first().equals(target) instead of the compareTo==0?